plugins/nexus-script-plugin/src/main/java/org/sonatype/nexus/script/plugin/RepositoryApi.java

```diff
@@ -213,6 +213,27 @@ Repository createDockerHosted(final String name,
                                 final boolean strictContentTypeValidation,
                                 final WritePolicy writePolicy);
 
+  /**
+   * Create a Docker hosted repository.
+   * @param name The name of the new Repository
+   * @param httpPort The http port to accept traffic for this Repository on (optional)
+   * @param httpsPort The https port to accept traffic for this Repository on (optional)
+   * @param blobStoreName The BlobStore the Repository should use
+   * @param v1Enabled Whether or not this Repository supports Docker V1 format
+   * @param strictContentTypeValidation Whether or not the Repository should enforce strict content types
+   * @param writePolicy The {@link WritePolicy} for the Repository
+   * @param forceBasicAuth whether or not this Repository should support anonymous pull
```

Comment:
This comment is not entirely true.

forceBasicAuth=true means use BASIC auth.
forceBasicAuth=false means use bearer token auth.

False is required for anonymous access but it doesn't guarantee it, also you could configure the permissions so that it isn't just "pull" specific. How about something like "whether to force basic auth (false is required to enable token auth which can be used for anonymous access)"

Reply:
@jlstephens89 thanks for remark. Fixed